src/main/user-impl/java/com/mysql/cj/jdbc/result/ResultSetImpl.java

```diff
@@ -841,7 +841,7 @@ public BigInteger getBigInteger(int columnIndex) throws SQLException {
             return new BigInteger(stringVal);
         } catch (NumberFormatException nfe) {
             throw SQLError.createSQLException(
-                    Messages.getString("ResultSet.Bad_format_for_BigInteger", new Object[] { Integer.valueOf(columnIndex), stringVal }),
+                    Messages.getString("ResultSet.Bad_format_for_BigInteger", new Object[] { stringVal, Integer.valueOf(columnIndex) }),
```

Comment:
Can we check the same for BigDecimal?

Reply:
We do not do a similar `getString` call for `getBigInteger` and we don't have a parameterized error for it either

Reply:
We do not do a similar `getString` call for `getBigInteger` and we don't have a parameterized error for it either